src/bp/core/services/dialog/flow/service.ts

```diff
@@ -101,6 +103,8 @@ export class FlowService {
         }
       })
 
+      // const orderedFlows = _.orderBy(flowsWithParents, x => x.name)
```

Comment:
Dead code :(

Reply:
Right, forgot to clean that when i changed the implementation